solr/core/src/test/org/apache/solr/cloud/hdfs/HdfsTestUtil.java

```diff
@@ -67,6 +68,14 @@ public static MiniDFSCluster setupClass(String dir, boolean safeModeTesting, boo
     LuceneTestCase.assumeFalse("HDFS tests were disabled by -Dtests.disableHdfs",
       Boolean.parseBoolean(System.getProperty("tests.disableHdfs", "false")));
 
+    // Checks that commons-lang3 FastDateFormat works with configured locale
```

Comment:
Currently certain locales like "ja-JP-u-ca-japanese-x-lvariant-JP" cause `FastDateFormat` from commons-lang3 to fail.

Reply:
Turns out only locale "ja-JP-u-ca-japanese-x-lvariant-JP" causes this failure. Emailed commons-user list about this. No response yet. I have not seen any failures in the past ~24 hours with this fix put in place.